ADRs/RH/SIG-SRE/ADR-00000 Template.md

```diff
@@ -0,0 +1,69 @@
+# ADR-00000 Template
+
+Authors: 
+
+
+## Status
+
+&lt;_Draft, Proposed, Reviewing, Accepted, Rejected, Superseded_> 
+
+
+## Problem Statement
+
+&lt;_Why does this ADR exist? What issue is it trying to solve?_>
+
+
+## Goals
+
+* &lt;_Clearly list the goals which this work will try to achieve._>
+* &lt;_The goals should be achievable, not visionary statements._>
+
+
+## Non-goals
+
+* &lt;_If any of the goals might be interpreted too broadly, here's where you limit their scope._>
+
+
+## Current Architecture
+
+* &lt;_If something is already solving the problem (even partially), here's where you mention it._>
+
+
+## Proposed Architecture
+
+* &lt;_Explain the full overview of the proposed solution.>
+* &lt;_Make it concise and simple; put diagrams; be concrete, avoid using vague generalizations._>
+
+
+## Challenges
+
+* &lt;_Call out any challenges here, e.g. upskilling needed, specific resources needed, very aggressive timelines if known, etc._>
+
+
+## Alternatives Considered
+
+* &lt;_Were any other approaches to solving the issue considered? If yes, describe them here._>
+* &lt;_Focus on the differences in outcomes (both positive and negative) between the proposed architecture and the alternatives._>
+
+
+## Dependencies
+
+* &lt;_Does this proposal depend on other things also being in place? List them._>
+
+
+## Stakeholders
+
+* &lt;_Who needs to sign off on this proposal for it to be implementable?_>
+* &lt;_Who should be informed, since they would be impacted?_>
```

Comment:
Our workflow assumes that gdocs are circulated internally with affected teams and some/all stakeholders for initial review before posting as a PR.  So I am not sure what we get by listing others explicitly.  People can always be '@'d on the github PR itself.

Reply:
Yup, makes sense, removing that point.